scripts/ros_install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo "Updating Locale..."
+sudo apt -qq install locales -y
+sudo locale-gen en_CA en_CA.UTF-8
+sudo update-locale LC_ALL=en_CA.UTF-8 LANG=en_CA.UTF-8
+export LANG=en_CA.UTF-8
+
+echo "Adding ROS source..."
+sudo apt -qq install software-properties-common
+sudo add-apt-repository universe
+
+sudo apt -qq install curl
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+
+echo "Updating sources and upgrading system packages..."
+sudo apt -qq update
+sudo apt -qq upgrade -y
+
+echo "Installing ROS... (will take a while)"
+sudo apt -qq install ros-foxy-desktop
```

Comment:
We actually want to install ros-[distro]-base instead

Reply:
Ok i wasnt sure anymore. Some people will need to installextra packages for autonomy stuff. 
Ill change it

Reply:
Ah fair enough. To address this, we could add a seperate script perhaps that would just install anymore required packages for autonomy. So that people won't need to install a ton of unnecessary packages to then only not use them if they don't do any autonomy work